lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/RunningWorkers.java

```diff
@@ -146,6 +158,13 @@ void launchTasklet(final Tasklet tasklet) {
         }
 
         final VortexWorkerManager vortexWorkerManager = runningWorkers.get(workerId.get());
+        if (tasklet.getAggregateFunctionId().isPresent() &&
+            !workerAggregateFunctionMap.containsKey(tasklet.getAggregateFunctionId().get())) {
```

Comment:
Could you confirm the argument of `containsKey()`? The `aggregateFunctionId` is an `integer`, whereas the map's key is `String`